### PR TITLE
Layout Changes

### PR DESCRIPTION
**Pull request by michalzy**

### CODE REVIEW

File: my-bulma-project.scss
Comment (alkeicam) [inline my-bulma-project.scss:1]: does this file is necessary here?

File: src/css/my-bulma-project.css
Comment (alkeicam) [inline src/css/my-bulma-project.css:1]: why is this here? seems it's not used at all

File: src/css/my-bulma-project.css.map
Comment (alkeicam) [inline src/css/my-bulma-project.css.map:1]: seems not used, please verify

File: src/css/style.css [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+@media (min-width: 1024px) {
+    .filters {
+        position: relative;
+        top: -40px;
+    }
+}
+
+
+.navbar {
+    background-color: transparent;
```
Comment (alkeicam) [inline src/css/style.css:14]: why don't you add dedicated selector instead of playing with app wide and framework wide selector

File: src/js/app-ui-components.js [hunk 2]
```diff
@@ -363,14 +370,13 @@
...
             </table>                
-        </div>
       `
               return template;
           },
         static: ['heading','when','metric', 'propA', 'propB', 'round'],
         // dynamic bound: 'errorMsg'
         initialize: function(el, data) {
-            
+            console.log(data);
```
Comment (alkeicam) [inline src/js/app-ui-components.js]: clear please

File: src/js/rivet-ext.js [hunk 3]
```diff
@@ -340,6 +344,10 @@
     rivets.formatters.numberRoundDecimal = function (value, arg) {
         return parseFloat(value.toFixed(arg))
     }
+    
+    rivets.formatters.sliceToSeven = function(array) {
```
Comment (alkeicam) [inline src/js/rivet-ext.js]: make it parametrized (array, size)

**Review by alkeicam — CHANGES_REQUESTED** [review]

File: my-bulma-project.scss
Comment (michalzy) [reply my-bulma-project.scss:1]: @alkeicam this one is necessary, it's overiding default bulma classes for new layout

File: src/css/my-bulma-project.css
Comment (michalzy) [reply src/css/my-bulma-project.css:1]: @alkeicam it's used in index.html   <link rel="stylesheet" href="css/my-bulma-project.css" />

Comment (michalzy) [reply src/css/my-bulma-project.css:1]: @alkeicam `  <link rel="stylesheet" href="css/my-bulma-project.css" />`

File: src/css/my-bulma-project.css.map
Comment (michalzy) [reply src/css/my-bulma-project.css.map:1]: @alkeicam Sass script generates source map files (.css.map) to facilitate a development feature that allows you to see your original source (Sass files) in the browser's developer tools instead of the compiled CSS. I would leave that there untouched.

File: src/js/rivet-ext.js (hunk 3 above)
Comment (michalzy) [reply src/js/rivet-ext.js]: @alkeicam done

File: src/js/app-ui-components.js (hunk 2 above)
Comment (michalzy) [reply src/js/app-ui-components.js]: @alkeicam done

File: src/css/style.css (hunk 1 above)
Comment (michalzy) [reply src/css/style.css:14]: @alkeicam done

File: src/css/my-bulma-project.css
Comment (alkeicam) [reply src/css/my-bulma-project.css:1]: rename to q247-styles.css

File: my-bulma-project.scss
Comment (alkeicam) [reply my-bulma-project.scss:1]: rename to q247-styles.css

Comment (michalzy) [reply my-bulma-project.scss:1]: @alkeicam done

File: src/css/my-bulma-project.css
Comment (michalzy) [reply src/css/my-bulma-project.css:1]: @done
